apps/spruce/src/hooks/useBetaFeatures/index.ts

```diff
@@ -10,53 +10,61 @@ import { ADMIN_BETA_FEATURES, USER_BETA_FEATURES } from "gql/queries";
 
 /**
  * `useAdminBetaFeatures` returns the beta features defined at the admin level.
- * @returns admin beta features
+ * @returns an object containing
+ * - betaFeatures: admin beta features
+ * - loading: boolean representing query loading state
  */
 export const useAdminBetaFeatures = () => {
-  const { data: adminSettingsData } = useQuery<
+  const { data, loading } = useQuery<
     AdminBetaFeaturesQuery,
     AdminBetaFeaturesQueryVariables
   >(ADMIN_BETA_FEATURES);
-  const { spruceConfig } = adminSettingsData ?? {};
+  const { spruceConfig } = data ?? {};
   const { ui } = spruceConfig ?? {};
   const { betaFeatures } = ui ?? {};
-  return betaFeatures;
+  return { betaFeatures, loading };
 };
 
 /**
  * `useUserBetaFeatures` returns the beta features defined at the user level.
- * @returns user's beta features
+ * @returns an object containing
+ * - betaFeatures: user beta features
+ * - loading: boolean representing query loading state
  */
 export const useUserBetaFeatures = () => {
```

Comment:
Not to be nitpicky, but what do you think of calling this `userBetaSettings`? I feel like that may help to distinguish between the top level features vs a user's individual choices.